dependency-reduced-pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.iota</groupId>
   <artifactId>iri</artifactId>
   <name>IRI</name>
-  <version>1.4.1.7</version>
+  <version>1.4.2.1</version>

```

Comment:
It is independent change.

Reply:
When I built it to check if my changes worked properly, I think it made this change automatically. See #523